app/services/admin/collections_report_query.rb

```diff
@@ -31,7 +31,7 @@ def generate
     def add_status_filter
       return if collections_report.statuses.empty?
 
-      self.query = query.where(head: { state: collections_report.statuses })
+      self.query = query.where(collection_versions: { state: collections_report.statuses })
```

Comment:
will this pull in all versions rather than just the head version?  Is this following the right relationship?

Reply:
You're right.
```
SELECT "collections".* FROM "collections" INNER JOIN "collection_versions" ON "collection_versions"."id" = "collections"."head_id" INNER JOIN "users" ON "users"."id" = "collections"."creator_id" WHERE "collection_versions"."state" = $1 ORDER BY collection_versions.name ASC
```

Reply:
Oh, no wait: That is correct: `INNER JOIN "collection_versions" ON "collection_versions"."id" = "collections"."head_id"`

Reply:
Thank you for reassuring me.